src/test/java/it/gov/pagopa/payhub/activities/mapper/ingestionflow/debtpositiontypeorg/DebtPositionTypeOrgMapperTest.java

```diff
@@ -0,0 +1,37 @@
+package it.gov.pagopa.payhub.activities.mapper.ingestionflow.debtpositiontypeorg;
+
+import it.gov.pagopa.payhub.activities.dto.ingestion.debtpositiontypeorg.DebtPositionTypeOrgIngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.util.TestUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.co.jemos.podam.api.PodamFactory;
+
+import static it.gov.pagopa.payhub.activities.util.TestUtils.checkNotNullFields;
+
+@ExtendWith(MockitoExtension.class)
+class DebtPositionTypeOrgMapperTest {
+
+  @InjectMocks
+  private DebtPositionTypeOrgMapper debtPositionTypeMapper;
+
+  private final PodamFactory podamFactory = TestUtils.getPodamFactory();
+
+
+  @Test
+  void givenDebtPositionTypeIngestionFlowFileWithPodamFactoryWhenMapThenOk() {
+    DebtPositionTypeOrgIngestionFlowFileDTO dto = podamFactory.manufacturePojo(
+            DebtPositionTypeOrgIngestionFlowFileDTO.class);
+    Long debtPositionTypeId = 123L;
+    Long organizationId = 456L;
+
+    var result = debtPositionTypeMapper.map(dto, debtPositionTypeId, organizationId);
+
+    Assertions.assertNotNull(result);
+    checkNotNullFields(result, "creationDate", "updateDate", "updateOperatorExternalId",
+        "updateTraceId", "debtPositionTypeOrgId","serviceId","notifyOutcomePushOrgSilServiceId","amountActualizationOrgSilServiceId");
```

Comment:
why should `serviceId`,`notifyOutcomePushOrgSilServiceId`,`amountActualizationOrgSilServiceId` be null?
you should map also them!

Could `serviceId` (IO service ID) be `codiceServizio`?

You should search OrgSilService to fill both `notifyOutcomePushOrgSilServiceId`,`amountActualizationOrgSilServiceId`: I see you have `codServNotificaEsitoPush` and `codServAttualizzazioneImporto`: on OrgSilService import where are you storing these info?

CC: @marcomatteuccieng